From ed3710517fd935537059543f808c343079708ec8 Mon Sep 17 00:00:00 2001
From: Giuseppe CAVALLARO <peppe.cavallaro@st.com>
Date: Wed, 17 Sep 2008 14:38:54 +0200
Subject: [PATCH] Export TMU2 symbols

Export symbols to have the stmmac built as dynamic module
when the TMU channel 2 is used as timer.

Signed-off-by: Giuseppe Cavallaro <peppe.cavallaro@st.com>
Signed-off-by: Stuart Menefy <stuart.menefy@st.com>
---
 arch/sh/kernel/timers/timer-tmu.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/arch/sh/kernel/timers/timer-tmu.c b/arch/sh/kernel/timers/timer-tmu.c
index 90d034c..a4d9fd9 100644
--- a/arch/sh/kernel/timers/timer-tmu.c
+++ b/arch/sh/kernel/timers/timer-tmu.c
@@ -251,6 +251,7 @@ int tmu2_register_user(void *fnt, void *data)
 
 	return 0;
 }
+EXPORT_SYMBOL(tmu2_register_user);
 
 void tmu2_unregister_user(void)
 {
@@ -259,6 +260,7 @@ void tmu2_unregister_user(void)
 
 	return;
 }
+EXPORT_SYMBOL(tmu2_unregister_user);
 
 static struct irqaction tmu2_irq = {
 	.name		= "TMU2 periodic timer",
-- 
1.5.3.6

